### ISSUE TITLE
Generally , using an single A100 (80G) gpu,  how long does it usually take for the model to answer one user question?

### ISSUE THREAD

**Issue by bigmancomeon** [defog-ai/sqlcoder#37]

**Comment by bigmancomeon** [comment]
> Once the model is loaded and the logits are initialized, ~5 seconds.

Can it take less time if I use  2* A100（80G）gpu?

**Comment by rishsriv** [comment]
No, but it'll be ~3x faster if you use an H100. Also, the 5 seconds is a high estimate for very complex queries. We get around ~1.9 seconds per query on average on the [eval dataset here](https://github.com/defog-ai/sql-eval)

**Comment by bigmancomeon** [comment]
> No, but it'll be ~3x faster if you use an H100. Also, the 5 seconds is a high estimate for very complex queries. We get around ~1.9 seconds per query on average on the [eval dataset here](https://github.com/defog-ai/sql-eval)

Got it,Thank you! But whether it's the A100 or the H100, they're both a bit expensive. Have you used other gpu and the model running speed is not slow?
